tests/scenarios/test_generic_nimbus.py

```diff
@@ -24,6 +24,14 @@ def test_experiment_unenrolls_via_about_studies_page():
     pass
 
 
+@scenario(
+    "../features/generic_nimbus.feature",
+    "The experiment can be unenrolled via opting out from studies",
+)
+def test_experiment_unenrolls_via_opting_out_of_studies():
+    pass
```

Comment:
I was going to say this looks suspicious, but looks like its prior art and I guess all the code is in the scenario's .feature file. 👍